security/Vidalia/Portfile

```diff
@@ -57,7 +57,8 @@ if {${os.arch} eq "arm"} {
     livecheck.regex The most recent stable release is: (\[\\d\.\]+)
 }
 
-depends_lib-append  port:tor
+# Accommodate tor-devel:
+depends_lib-append  path:bin/tor:tor
```

Comment:
For future reference, this can also be written as `bin:tor:tor`

Reply:
@herbygillot Done